mptt/models.py

```diff
@@ -182,6 +182,11 @@ def insertion_target_filters(self, instance, order_insertion_by):
                 field = instance._meta.get_field(field_name)
                 value = field.pre_save(instance, True)
 
+            # not really sure how this works, but fixes #340 
```

Comment:
We can't merge code that we don't understand. I think this needs a comment properly explaining what it's actually doing.